client/src/components/Layout/GamePage.jsx

```diff
@@ -18,14 +18,20 @@ const GamePage = (props) => {
   const {
     render, classes, startGame, score, status,
   } = props;
+
+  const buttonMessage = () => {
+    if (status === 0) return 'Start';
+    if (status === 1) return 'Pause';
+    return 'Restart';
```

Comment:
one more should be 'game ended'

Reply:
There's Restart instead of Game Ended for the User. Hasan is going to do further changes to the logic for the Modal. This is the boilerplate logic.